src/js/control-bar/text-track-controls/chapters-button.js

```diff
@@ -90,11 +90,13 @@ class ChaptersButton extends TextTrackButton {
     let menu = this.menu;
     if (menu === undefined) {
       menu = new Menu(this.player_);
-      menu.contentEl().appendChild(Dom.createEl('li', {
+      let title = Dom.createEl('li', {
```

Comment:
1. It's unlikely that it should be a problem, especially since it should be the first item.
2. I'll have to take a look at that and see whether it applies in this case or what its used for. Thanks.